src/main/scala/pureconfig/StringConvert.scala

```diff
@@ -33,3 +34,10 @@ object StringConvert {
   implicit val readShort = fromUnsafe[Short](_.toShort, _.toString)
   implicit val readString = fromUnsafe[String](identity, identity)
 }
+
+/**
+ * Implicit [[StringConvert]] instances defined such that they can be overriden by library consumer via a locally defined implementation.
+ */
+trait LowPriorityStringConvertImplicits {
+  implicit val readURL = StringConvert.fromUnsafe[URL](new URL(_), _.toString)
```

Comment:
@melrief Yeah, I definitely think that's the right thing to do. However, I think if it's done, it should be done as with an accompanying version bump that indicates a possible backward compatibility break.

I'd be happy to send another PR for that if you like.